controller/workitem.go

```diff
@@ -568,7 +569,7 @@ func getVersion(version interface{}) (int, error) {
 // []string object containing a set of CSV formatted data lines and a header line with labels.
 // This methods combines all CSV data of all WITs into a single CSV and returns the CSV as well
 // as the field headers as a seperate slice
-func ConvertWorkItemsToCSV(ctx context.Context, app application.Application, allWits []workitem.WorkItemType, wis []workitem.WorkItem, includeHeaderLine bool) (string, []string, error) {
+func ConvertWorkItemsToCSV(ctx context.Context, db application.DB, allWits []workitem.WorkItemType, wis []workitem.WorkItem, childLinks link.WorkItemLinkList, parents link.AncestorList, idNumberMap *map[string]string, includeHeaderLine bool) (string, []string, error) {
```

Comment:
@michaelkleinhenz For maps, we don't need to use pointers. 
See https://play.golang.org/p/B-rxxATsxBX. When you pass in a map to a function **which is already initialized**, the map behaves as if it was passed by reference.

See also https://dave.cheney.net/2017/04/30/if-a-map-isnt-a-reference-variable-what-is-it

Reply:
Fixed a3f3033.